glean-core/rlb/src/lib.rs

```diff
@@ -320,6 +320,17 @@ pub fn shutdown() {
     }
 }
 
+/// Block on the dispatcher emptying.
+///
+/// This will panic if called before Glean is initialized.
+fn block_on_queue() {
```

Comment:
Since this is now living in the RLB lib, what do you think about giving it a more specific name? For example `block_on_dispatcher`?

Reply:
works for me.